acceptance/suites/pre_suite/foss/95_install_pdb.rb

```diff
@@ -2,7 +2,6 @@
 matching_puppetdb_platform = puppetdb_supported_platforms.select { |r| r =~ master.platform }
 skip_test unless matching_puppetdb_platform.length > 0
 
-require 'json'
```

Comment:
Why this change?

Reply:
oh i should have pulled it into a different commit, but it isn't used in that test.